helm/reana/README.md

```diff
@@ -95,6 +95,14 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `secrets.gitlab.REANA_GITLAB_OAUTH_APP_ID`               | GitLab OAuth application id                                                          | None                                            |
 | `secrets.gitlab.REANA_GITLAB_OAUTH_APP_SECRET`           | **[Do not use in production, use secrets instead]** GitLab OAuth application secret  | None                                            |
 | `secrets.reana.REANA_SECRET_KEY`                         | **[Do not use in production, use secrets instead]** REANA encryption secret key      | None                                            |
+| `sso.enabled`                                            | Enable and configure your own third-party authentication service                     | false                                           |
```

Comment:
What about keeping `sso.*` values for the old `cern_openid` technique, and introducing `keycloak_*` values for the new Keycloak based way?

(And we'll deprecate `sso.*` in the future.)

Reply:
I changed the naming to `keycloak*` and moved the key and secret to `secrets.keycloak` as well.